src/pages/Main/Banners/Banners.js

```diff
@@ -0,0 +1,46 @@
+import React, { useState } from 'react';
+import BannerList from './BannerList';
+
+const Banners = props => {
+  const [currentSlide, setCurrentSlide] = useState(1);
+  const handleSlide = (e, direction) => {
+    const newCurrentSlide =
+      direction === 'left' ? currentSlide - 1 : currentSlide + 1;
+    direction === 'left'
+      ? setCurrentSlide(currentSlide === 1 ? 3 : newCurrentSlide)
+      : setCurrentSlide(currentSlide === 3 ? 1 : newCurrentSlide);
+  };
+  return BANNER.map(banner => (
+    <BannerList
+      banner={banner}
+      key={banner.idx}
+      handleSlide={handleSlide}
+      currentSlide={currentSlide}
+    />
+  ));
```

Comment:
```suggestion
const Banners = props => {
  const [currentSlide, setCurrentSlide] = useState(1);
  
  const handleSlide = (e, direction) => {
    const newCurrentSlide =
      direction === 'left' ? currentSlide - 1 : currentSlide + 1;
      
    direction === 'left'
      ? setCurrentSlide(currentSlide === 1 ? 3 : newCurrentSlide)
      : setCurrentSlide(currentSlide === 3 ? 1 : newCurrentSlide);
  };
  
  return BANNER.map(banner => (
    <BannerList
      banner={banner}
      key={banner.idx}
      handleSlide={handleSlide}
      currentSlide={currentSlide}
    />
  ));
```
- 구분되는 로직들은 한줄씩 띄워주시면 더 보기 좋을거 같습니다
- 지금은 일단 슬라이드의 시작과 끝이 1과 3으로 하드코딩 되어 있는데 1은 그렇더라도 3은 하드코딩 되면 안될거 같아요

Reply:
줄 바꿈 통해서 가독성 늘렸고 하드코딩 했던 부분은 우선 BANNER.length 이용해서 변경하였습니다!